ccm_web/client/src/App.tsx

```diff
@@ -92,6 +92,10 @@ function App (props: AppProps): JSX.Element {
           </Switch>
         </Router>
         <ConsumerTest ltiKey={props.ltiKey} />
+        {
+          globals?.environment === 'development' && props.ltiKey !== undefined &&
+            <Link href={`/swagger?token=${props.ltiKey}`}>Swagger UI</Link>
```

Comment:
This is smart. Somewhere down the lane we need to have a footer and place this URL in it. @chrisrrowland what do you think?

Reply:
Are you hinting at something different than what @ssciolla  has already done here?  I guess you mean a more formal footer?

We could, especially if we want a footer for other purposes.  Otherwise I also think it's fine just tagging it on like this in development.

Reply:
We will probably want a footer to put a copyright message and maybe version information, and this link could live there. It may also just be a good UI design thing, lets people know they're at the end of the page and prevents important content from crowding up near the window edge. Separate issue though.